internal/services/applications/application_data_source.go

```diff
@@ -56,6 +63,21 @@ func applicationDataSource() *schema.Resource {
 				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"accept_mapped_claims": {
```

Comment:
should we use this for consistency with other bools?
```suggestion
						"mapped_claims_enabled": {
```